apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/Manage/index.tsx

```diff
@@ -23,13 +23,22 @@ import { TrustchainNotFound } from "@ledgerhq/trustchain/errors";
 
 const WalletSyncManage = () => {
   const { t } = useTranslation();
+  const [forcedTimeLoader, setForcedTimeLoader] = useState(true);
```

Comment:
Can we extract the logic regarding this timer in a separate hook ?

Reply:
Done ✅